posts/call-for-maintainers-2020.md

```diff
@@ -0,0 +1,17 @@
+**We're looking for new Track Maintainers. Interested? Read on!**
+
+Exercism is powered by the energy of many volunteers, especially the [track maintainers](https://exercism.io/team/maintainers) who are responsible for creating and managing our Language Tracks.
+
+We're about to start a big push towards version 3 (v3) of Exercism. Getting all the tracks ready for v3 is going to take a big community effort, and we're looking to add a few more maintainers to oversee and manage that task.
+
+In the push to v3, our maintainers will be responsible for:
+- **Designing the track:** Working out what someone needs to learn to become fluent in a language, and mapping this out as a pathway.
+- **Decision making:** Making design decisions around the tooling the track uses, the standards it conforms, the defaults it uses, etc.
+- **Encouraging and supporting contributions:** Providing a welcoming environment for people that want to help out, answering their questions, etc
+- **Reviewing PRs:** A lot of the acutal work of building out v3 tracks will be done by the community. Maintainers act as gatekeepers, reviewing the work that the community does and ensuring the track's standards are enforced.
+
+This might all sound a bit daunting but our more experienced team will help you get started and hold your hand as you learn the ropes. Ideally you need to be able to commit 3hrs per week (preferably spread over a week) for about 6 months, although obviously life ebs and flows, and some weeks might work better than others. You'll also prefably have some experience of Exercism - either as a mentor or having contributed to the codebase. This isn't strictly a requirement, but it's nice to know you've cared about Exercism for a while.
+
+Maintainers are volunteers so don't get paid. The key motiviation will ideally be you wanting to help a non-for-profit organisation, but being an Exercism maintainer also looks pretty good on a Resume if that's something that matters to you.
```

Comment:
Maybe the focus should be less about helping a non-profit organisation, but more about the students Exercism helps learn?

Reply:
We could maybe also add additional benefits such as working with clever people and learning tons of things?

Reply:
Agree and agree.

Reply:
```suggestion
Maintainers are volunteers so don't get paid. The key motivation will ideally be you wanting to help people develop their programming skills, and help a non-for-profit organisation. Being an Exercism maintainer also a great opportunity to work with some fun, smart people, learn a lot of things, and it looks pretty good on a Resume if that's something that matters to you.
```